bootstrap/command.go

```diff
@@ -58,6 +61,27 @@ func init() {
 	flag.BoolVar(&runGoTests, "t", false, "build and run go tests during bootstrap")
 }
 
+func listFiles(srcDir string) (filePaths []string, err error) {
+	listFile := ModulesFile
```

Comment:
Take listFile as a parameter so that only Main is referencing the global variables

Reply:
thanks